<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgba(255, 255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>LogIn</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="IdEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>241</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
   </property>
   <property name="placeholderText">
    <string>Uesr Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnlogin">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한컴 고딕</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btnlogin{
	background-color: rgba(85, 98, 112, 255);
	color:rgba(255, 255, 255, 200);
	border-radius:5px;
}

QPushButton#btnlogin:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}

QPushButton#btnlogin::pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(200, 200, 200, 255);
}</string>
   </property>
   <property name="text">
    <string>L o g i n</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>241</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="coopIdEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>241</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Coop Name</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
